docs/api.md

```diff
@@ -0,0 +1,376 @@
+    </Surface>
+  );
+}
+```
+
+## Shape
+
+Used to draw arbitrary vector shapes from Path. Shape implements Transform as a mixin which means it has all transform methods available for moving, scaling and rotating a shape.
+
+|       Prop        |              Type               |  Default  |
+| :---------------: | :-----------------------------: | :-------: |
+|  ...opacityProps  |         `OpacityProps`          |    ---    |
+| ...transformProps |        `TransformProps`         |    ---    |
+|       fill        |        `string \| Brush`        |    ---    |
+|      stroke       |            `string`             |    ---    |
+|     strokeCap     | `'butt' \| 'square' \| 'round'` | `'round'` |
+|    strokeDash     |         `Array<number>`         |    ---    |
+|    strokeJoin     | `'miter' \| 'bevel' \| 'round'` | `'round'` |
+|    strokeWidth    |            `number`             |    `1`    |
+|         d         |        `string \| Path`         |    ---    |
+|       width       |            `number`             |    `0`    |
+|      height       |            `number`             |    `0`    |
+|     children      |          `React.Node`           |    ---    |
+
+```js
+import {Dimensions} from 'react-native';
+import {Surface, Shape} from '@react-native-community/art';
+
+const SURFACE_DIMENSIONS = Dimensions.get('window').width;
+
+function Heart() {
+  return (
+    <Surface width={SURFACE_DIMENSIONS} height={SURFACE_DIMENSIONS}>
+      <Shape
+        d="M 10,30 A 20,20 0,0,1 50,30 A 20,20 0,0,1 90,30 Q 90,60 50,90 Q 10,60 10,30 z"
+        fill="#d39494"
+      />
+    </Surface>
+  );
+}
+```
+
+## Text
+
+Text component creates a shape based on text content using native text rendering.
+
+|       Prop        |              Type               |  Default  |
+| :---------------: | :-----------------------------: | :-------: |
+| ...transformProps |        `TransformProps`         |    ---    |
+|  ...opacityProps  |         `OpacityProps`          |    ---    |
+|       fill        |        `string \| Brush`        |    ---    |
+|      stroke       |            `string`             |    ---    |
+|     strokeCap     | `'butt' \| 'square' \| 'round'` | `'round'` |
+|    strokeDash     |         `Array<number>`         |    ---    |
+|    strokeJoin     | `'miter' \| 'bevel' \| 'round'` | `'round'` |
+|    strokeWidth    |            `number`             |    `1`    |
+|       width       |            `number`             |    `0`    |
+|      height       |            `number`             |    `0`    |
+|     alignment     | `'center' \| 'left' \| 'right'` | `'left'`  |
+|       font        |        `string \| Font`         |    ---    |
+|       path        |        `string \| Path`         |    ---    |
+|     children      |    `string \| Array<string>`    |    ---    |
+
+## Path
+
+Generate an SVG `path` that you can pass to the `Shape` element.
+
+`constructor`
+
+```js
+new Path(path: string | Path)
+```
+
+`Path.move`
```

Comment:
I'd prefer using headings, it's gonna be easier for navigation and link to certain methods
```suggestion
### `move`
```